state/upgrades_test.go

```diff
@@ -100,7 +100,7 @@ func (s *upgradesSuite) TestStripLocalUserDomainModels(c *gc.C) {
 	delete(initialModel, "txn-revno")
 	initialModel["owner"] = "test-admin"
 
-	// TODO (mattyw, cmars) We need to confirm that setting sla in this test
+	// TODO (mattyw, cmars) We need to confirm that setting sla and meter-status in this test
```

Comment:
@mattyw Thanks for the explanation. I think we can remove this TODO then as this test is unrelated and we're just testing for consistent model migration behavior.